### ISSUE TITLE
How to pass same filtered SeriesList from two different days to asPercent()

### ISSUE THREAD

**Issue by ssinhaRepo** [graphite-project/graphite-web#2263]
I want to pass the same filtered seriesList of a metric as a numerator and denominator to asPercent(), can't seems to figure how, please help. Below is my use case:

 I have a graphite metric for event counts for different placements as following:

    stats.counters.events.placements.count

I am interested to find the percentage comparison between today's and yesterdays's same time 2 hour moving averages for each placement. I have the following function which is working fine:

    asPercent(
        groupByNode(
            movingAverage(stats.counters.events.*.count, '2hour'),
        3, "averageSeries"),
        groupByNode(
            movingAverage(
                timeShift(stats.counters.events.*.count, '1d'),
            '2hour'),
        3, "averageSeries")
    )

Now I want to enhance it, by adding some filter. E.g. some of my placement counts are very low, I want to filter those placements out for which yesterday's movingAverage count was less than 100.

so basically find the filtered seriesList of placements from yesterday (denominator) and compare the SAME placement list counts from today (numerator)

I got yesterday's list:

    averageAbove(timeShift(stats.counters.events.*.count, '1d'), 100)

but not sure how to enforce the same placement list for today's count (on the numerator), since applying averageAbove() on today's count i.e.

    averageAbove(stats.counters.events.*.count, 100)

may give a different list, and pass two different seriesLists to the asPercent().

Thanks in advance.

**Comment by DanCech** [comment]
If you are using 1.1.2 you can provide a set of nodes to `asPercent` that it will use to match the series.

http://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.asPercent

**Comment by ssinhaRepo** [comment]
Is there anyway to do the whole thing using mapSeries()/reduceSeries()?

**Comment by ssinhaRepo** [comment]
@DanCech, as I mentioned above we can't hardcode the nodes since the new nodes can be added in future. However, I do know the list of nodes/placements which I want to exclude, is there any blacklisting functionality in graphite like !{node1, node2...}
